### ISSUE TITLE
v0.5.0-rc3 docker-machine create --driver digitalocean

### ISSUE THREAD

**Issue by outtolunc** [docker/machine#2143]
This issue still exists, and has been stated 'this will be fixed in v0.5.0 , will it?  

If nothing else have it fall-through to 
/etc/os-release
/etc/redhat-release
/etc/centos-release
etc..

Running pre-create checks...
Creating machine...
Waiting for machine to be running, this may take a few minutes...
Machine is running, waiting for SSH to be available...
Detecting operating system of created instance...
Error creating machine: Error detecting OS: Error getting SSH command: Something went wrong running an SSH command!
command : cat /etc/os-release
err     : exit status 1
output  : cat: /etc/os-release: No such file or directory

**Comment by nathanleclaire** [comment]
I don't think it will be fixed in the 0.5.0 release.  Where did you read that?  Is there a duplicate issue we can consolidate into?

**Comment by outtolunc** [comment]
google:  docker-machine /etc/os-release issue

not sure which had the 'will be fixed in v0.5.0 was in.. as there are so
many.

1031 waiting for vm to start
1684 ..

1643
1596
1019
1833
1591

...

**Comment by nathanleclaire** [comment]
At a quick glance I don't see any issues which clearly reference this bug in the name but I'll treat this one as the bug report until / if we find such a dupe.

**Comment by nathanleclaire** [comment]
But for reference, it is still an issue in 0.5.0.

**Comment by Diaoul** [comment]
I'm affected by this too

    $ cat /etc/redhat-release
    Red Hat Enterprise Linux Server release 6.6 (Santiago)

**Comment by jbaramidze** [comment]
still no solution for it?

**Comment by harishanchu** [comment]
+1

**Comment by yawn** [comment]
If I understand [this](http://0pointer.de/blog/projects/os-release.html) summary correctly, `/etc/os-release` is systemd-specific which does not apply to CentOS 6.

**Comment by xlDoug** [comment]
+1

**Comment by nioperas06** [comment]
+1

**Comment by ericraio** [comment]
+1

**Comment by andrzejmartyna** [comment]
The same with macOS which does not have any "*release" or "*version" files in "/etc".
Some unanswered question also here:
https://forums.docker.com/t/connecting-to-docker-running-on-mac-server/9174

**Comment by mnesov** [comment]
Same problem.
Host: MacOS 10.15.7
docker-machine version 0.16.2, build bd45ab13
```
Running pre-create checks...
Creating machine...
(mini) Importing SSH key...
Waiting for machine to be running, this may take a few minutes...
Detecting operating system of created instance...
Waiting for SSH to be available...
Detecting the provisioner...
Error creating machine: Error detecting OS: Error getting SSH command: ssh command error:
command : cat /etc/os-release
err     : exit status 1
output  : cat: /etc/os-release: No such file or directory
```

**Comment by cianiandreadev** [comment]
Same Problem on MacOS 11.2.3
Any solutions?
